src/zip.js

```diff
@@ -8,6 +8,11 @@ module.exports = function(gj, options) {
     var zip = new JSZip(),
         layers = zip.folder(options && options.folder ? options.folder : 'layers');
```

Comment:
it will create empty folder names 'layers' this line should be placed in the `else` block of line 12 if statement